### ISSUE TITLE
[artifacts-centos8-arm]: Failing due to hard coded manager version which doesn't support arm

### ISSUE THREAD

**Issue by yaronkaikov** [scylladb/scylla-cluster-tests#4288]
In 2021.1.8 we added arm support, since SCT are using manager 2.3 for this version those arm based artifacts are failig

https://jenkins.scylladb.com/job/enterprise-2021.1/job/artifacts/job/artifacts-centos8-arm-test/8/console

**Comment by fgelcer** [comment]
> In 2021.1.8 we added arm support, since SCT are using manager 2.3 for this version those arm based artifacts are failig
> 
> https://jenkins.scylladb.com/job/enterprise-2021.1/job/artifacts/job/artifacts-centos8-arm-test/8/console

@yaronkaikov , from which version of the manager ARM is supported?

**Comment by Orenef11** [comment]
The run failed because of:
```
09:14:07  Command: 'sudo yum install -y scylla-manager-agent'
09:14:07  
09:14:07  Exit code: 1
09:14:07  
09:14:07  Stdout:
09:14:07  
09:14:07  Scylla Manager for Centos - aarch64             709  B/s | 353  B     00:00    
09:14:07  
09:14:07  Stderr:
09:14:07  
09:14:07  Errors during downloading metadata for repository 'scylla-manager-2.3':
09:14:07    - Status code: 404 for http://downloads.scylladb.com/downloads/scylla-manager/rpm/centos/scylladb-manager-2.3/aarch64/repodata/repomd.xml (IP: 52.217.204.109)
09:14:07  Error: Failed to download metadata for repo 'scylla-manager-2.3': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried
```

**Comment by yaronkaikov** [comment]
@fgelcer 2.6

**Comment by fgelcer** [comment]
ping @ShlomiBalalis

**Comment by rayakurl** [comment]
@roydahan @ShlomiBalalis  can you please explain what is needed from our side? Thanks

**Comment by fruch** [comment]
> @roydahan @ShlomiBalalis can you please explain what is needed from our side? Thanks

To backport the needed fixes/changes to get manager 2.6 working on sct branch 2021.1

**Comment by roydahan** [comment]
@ShlomiBalalis please send a PR to change the manager version to 2.6 in 2021.1 (it still uses the explicit repo).

If it doesn't work or there is an issue, let's disable the use of manager for now in these tests.
